interfaces/Portalicious/src/app/pages/projects-overview/project-summary-card/project-summary-card.component.html

```diff
@@ -0,0 +1,110 @@
+<ng-template
+  #metric
+  let-value="value"
+  let-label="label"
+>
+  <dd class="px-2 py-1 font-bold">
+    @switch (metrics.status()) {
+      @case ('pending') {
+        <p-skeleton
+          class="my-1 cursor-progress text-lg"
+          height="1.125rem"
+          [width]="randomWidth"
+        />
+      }
+      @case ('success') {
+        {{ value }}
+      }
+    }
+  </dd>
+  <dt class="px-2 py-1 text-sm">
+    {{ label }}
+  </dt>
+</ng-template>
+
+<p-card
+  styleClass="group relative h-full border border-transparent *:h-full focus-within:border-primary hover:border-primary [&_.p-card-title]:flex-grow [&_.p-card-content]:flex-end"
+>
```

Comment:
This property doesn't get automatically formatted by the Tailwind-VSCode-extention... But if your rename it shortly to `class`, it does!

So this needs a proper fix somewhere in the Tailwind-config maybe, but for now its "doable"... :/ maybe...

Reply:
Addressed here https://github.com/global-121/121-platform/pull/5659